.devcontainer/post-create.sh

```diff
@@ -4,10 +4,11 @@ chmod +x /workspace/comfystream/docker/entrypoint.sh
 cd /workspace/comfystream
 
 # Install Comfystream in editable mode.
-/workspace/comfystream/docker/entrypoint.sh --editable
+echo -e "\e[32mInstalling Comfystream in editable mode...\e[0m"
+/workspace/miniconda3/envs/comfystream/bin/python3 -m pip install -e . > /dev/null
 
 # Create a symlink to the entrypoint script.
-echo 'alias prepare_examples="/workspace/comfystream/docker/entrypoint.sh --download-models --build-engines"' >> ~/.bashrc
+echo 'alias prepare_examples="/workspace/comfystream/.devcontainer/entrypoint.sh --download-models --build-engines"' >> ~/.bashrc
```

Comment:
@eliteprox I'm fine with keeping it in docker but now have it closer to the container that uses it.

Reply:
I think this makes more sense in the docker folder, but it's a small nit